sources/api/migration/migrations/v1.10.0/dns-settings/src/main.rs

```diff
@@ -0,0 +1,25 @@
+#![deny(rust_2018_idioms)]
+
+use migration_helpers::common_migrations::AddPrefixesMigration;
+use migration_helpers::{migrate, Result};
+use std::process;
+
+/// We added new settings under `settings.kernel.modules` for configuring
```

Comment:
The comment doesn't match what the migration is changing, it should be something like `settings.dns.*`